cmd/tailscaled/tailscaled.go

```diff
@@ -788,6 +795,35 @@ func beChild(args []string) error {
 	return f(args[1:])
 }
 
+var serveTailfsFunc = serveTailfs
+
+// serveTailfs serves one or more tailfs on localhost using the WebDAV
+// protocol. On UNIX and MacOS tailscaled environment, tailfs spawns child
+// tailscaled processes in serve-tailfs mode in order to access the fliesystem
+// as specific (usually unprivileged) users.
+//
+// serveTailfs prints the address on which it's listening to stdout so that the
+// parent process knows where to connect to.
+func serveTailfs(args []string) error {
+	if len(args) < 2 {
```

Comment:
`len(args) == 1` is a line 812 [error case](https://github.com/tailscale/tailscale/blob/b23e57805f95074041513e3c21e4b1abcff0a0da/cmd/tailscale/cli/share.go#L163-L165)

Reply:
Good point, thanks.